src/auth/SecureRemotePassword/server/SrpServer.cpp

```diff
@@ -274,6 +274,10 @@ int SrpServer::authenticate(CheckStatusWrapper* status, IServerBlock* sb, IWrite
 
 			account = sb->getLogin();
 
+			const size_t len = strlen(account.c_str());
```

Comment:
Why not
const size_t len = account.length();

Reply:
Yes, that would be better